[2021-11-22 13:33:12,032] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: ELT-python-snowflake.datamodel_dimuser manual__2021-11-22T13:31:22.135157+00:00 [queued]>
[2021-11-22 13:33:12,048] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: ELT-python-snowflake.datamodel_dimuser manual__2021-11-22T13:31:22.135157+00:00 [queued]>
[2021-11-22 13:33:12,049] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-22 13:33:12,050] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-11-22 13:33:12,051] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-22 13:33:12,063] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): datamodel_dimuser> on 2021-11-22 13:31:22.135157+00:00
[2021-11-22 13:33:12,070] {standard_task_runner.py:52} INFO - Started process 4661 to run task
[2021-11-22 13:33:12,075] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ELT-python-snowflake', 'datamodel_dimuser', 'manual__2021-11-22T13:31:22.135157+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/ELT-python-snowflake.py', '--cfg-path', '/tmp/tmpei5e8i7t', '--error-file', '/tmp/tmpo83ob0vq']
[2021-11-22 13:33:12,076] {standard_task_runner.py:77} INFO - Job 83: Subtask datamodel_dimuser
[2021-11-22 13:33:12,222] {logging_mixin.py:109} INFO - Running <TaskInstance: ELT-python-snowflake.datamodel_dimuser manual__2021-11-22T13:31:22.135157+00:00 [running]> on host bd3f5d3b0fb3
[2021-11-22 13:33:12,333] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ELT-python-snowflake
AIRFLOW_CTX_TASK_ID=datamodel_dimuser
AIRFLOW_CTX_EXECUTION_DATE=2021-11-22T13:31:22.135157+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-22T13:31:22.135157+00:00
[2021-11-22 13:33:12,336] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-11-22 13:33:12,338] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python3 /opt/***/elt/user-orders/transform-snowflake/6_enrich_to_datamodel/extract_to_dimuser.py']
[2021-11-22 13:33:12,362] {subprocess.py:85} INFO - Output:
[2021-11-22 13:33:16,410] {subprocess.py:89} INFO - WARNING: An illegal reflective access operation has occurred
[2021-11-22 13:33:16,411] {subprocess.py:89} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/spark-unsafe_2.12-3.1.1.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-11-22 13:33:16,413] {subprocess.py:89} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-11-22 13:33:16,414] {subprocess.py:89} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-11-22 13:33:16,415] {subprocess.py:89} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-11-22 13:33:17,010] {subprocess.py:89} INFO - 21/11/22 13:33:17 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-11-22 13:33:17,437] {subprocess.py:89} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-11-22 13:33:17,438] {subprocess.py:89} INFO - Setting default log level to "WARN".
[2021-11-22 13:33:17,440] {subprocess.py:89} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2021-11-22 13:33:25,925] {subprocess.py:89} INFO - /home/***/.local/lib/python3.7/site-packages/snowflake/connector/options.py:97: UserWarning: You have an incompatible version of 'pyarrow' installed (4.0.1), please install a version that adheres to: 'pyarrow<5.1.0,>=5.0.0; extra == "pandas"'
[2021-11-22 13:33:25,926] {subprocess.py:89} INFO -   _expected_pyarrow_version,
[2021-11-22 13:33:26,670] {subprocess.py:93} INFO - Command exited with return code 0
[2021-11-22 13:33:26,713] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=ELT-python-snowflake, task_id=datamodel_dimuser, execution_date=20211122T133122, start_date=20211122T133312, end_date=20211122T133326
[2021-11-22 13:33:26,762] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-22 13:33:26,814] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
